extensions/wikia/EditPageLayout/js/plugins/WikitextSyntaxHighlighter.js

```diff
@@ -451,90 +416,159 @@ define('WikiTextSyntaxHighlighter', ['wikia.window', 'wikia.document', 'wikia.lo
 			syncScrollY();
 		}
 		if (wpTextbox1.offsetHeight !== wpTextbox0.offsetHeight) {
-			wpTextbox0.style.height = wpTextbox1.offsetHeight + 'px';
+			var height = wpTextbox1.offsetHeight + "px";
+			wpTextbox0.style.height = height;
+			wpTextbox1.style.marginTop = "-" + height;
 		}
 	}
 
-	function setup (textarea) {
-		var focus,
-			scrollTop,
-			syntaxStyleElement,
-			textboxContainer,
-			wpTextbox1Style;
-
-		wpTextbox0 = document.createElement('div');
-		wpTextbox0.id = 'wpTextbox0';
-
-		wpTextbox1 = textarea;
-		wpTextbox1.id = 'wpTextbox1';
-		wpTextbox1.classList.add('highlighted');
-
-		syntaxHighlighterConfig.timeout = syntaxHighlighterConfig.timeout || 100;
-
-		textboxContainer = document.createElement('div');
-		syntaxStyleElement = document.createElement('style');
-		syntaxStyleTextNode = syntaxStyleElement.appendChild(document.createTextNode(''));
-
-		// The styling of the textbox and the background div must be kept very similar
-		wpTextbox1Style = window.getComputedStyle(wpTextbox1);
+	function setup(textarea) {
```

Comment:
This now accepts `texarea` fo backward compat. It was also using `syntaxHighlighterSiteConfig` which I removed because we don't have it